packages/wallets/phantom/src/adapter.ts

```diff
@@ -75,21 +75,48 @@ export class PhantomWalletAdapter extends BaseMessageSignerWalletAdapter {
             ? WalletReadyState.Unsupported
             : WalletReadyState.NotDetected;
 
+    isIosAndRedirectable() {
```

Comment:
Let's externalize this function. It doesn't access `this` so it shouldn't be a method (esp a public one). We could even move it to the base package, if we expect it to be reused by Solflare and other wallets.